main.py

```diff
@@ -11,49 +11,39 @@
 
 class Main:
     def __init__(self):
-        self.logger: logger = logger()
-        
-        self.server: server
-        
-        self.plugin_folder_path: str = os.path.join(os.getcwd(),"plugins")
-
-        self.loaded_plugins:list = []
-        
-        
+        self.plugin_folder_path: str = os.path.join(os.getcwd(), "plugins")
+        self.loaded_plugins: list = []
         
     def on_load(self):
-        self.logger.info("DevTools is on fire!!!")
+        self.server.logger.info("DevTools is on fire!!!")
```

Comment:
look at new commit

Reply:
yes

Reply:
reinitializing a new logger is pointless